modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/upgrade/registry/v_1_17/AssetVocabularyUpgradeProcess.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.content.dashboard.document.library.internal.upgrade.registry.v_1_17;
+
+import com.liferay.asset.kernel.model.AssetVocabulary;
+import com.liferay.asset.kernel.service.AssetVocabularyLocalService;
+import com.liferay.portal.kernel.repository.model.FileEntry;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
+import com.liferay.portal.kernel.service.CompanyLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portlet.asset.util.AssetVocabularySettingsHelper;
+
+import java.util.Arrays;
+
+/**
+ * @author Cristina González
+ */
+public class AssetVocabularyUpgradeProcess extends UpgradeProcess {
+
+	public AssetVocabularyUpgradeProcess(
+		AssetVocabularyLocalService assetVocabularyLocalService,
+		ClassNameLocalService classNameLocalService,
+		CompanyLocalService companyLocalService) {
+
+		_assetVocabularyLocalService = assetVocabularyLocalService;
+		_classNameLocalService = classNameLocalService;
+		_companyLocalService = companyLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		_companyLocalService.forEachCompany(
+			company -> {
+				AssetVocabulary audienceAssetVocabulary =
+					_assetVocabularyLocalService.fetchGroupVocabulary(
+						company.getGroupId(),
+						StringUtil.toLowerCase(
```

Comment:
@cgoncas, I think that we could avoid using StringUtil and GetterUtil and just use "audience or "stage" like we do in other places.

Reply:
Totally true